configs/aws.reference.conf

```diff
@@ -427,33 +435,58 @@ cloudera-manager {
     # MySQL limits usernames to sixteen characters. Therefore, limit usernamePrefix values for databases on MySQL to
     # seven characters; the remaining nine characters are used by the randomized suffix generated by Director.
     #
-
-    # databaseTemplates {
-    #     CLOUDERA_MANAGER {
-    #         name: cmtemplate
-    #         databaseServerName: mysql1 # Must correspond to an external database server named above
-    #         databaseNamePrefix: scm
-    #         usernamePrefix: cmadmin
-    #     }
+    # Note that the databaseServerName must correspond to an external database server named above, which
+    # can be the provisioned RDS instance or another existing database server.
     #
-    #     ACTIVITYMONITOR { ... }
-    #
-    #     REPORTSMANAGER { ... }
-    #
-    #     NAVIGATOR { ... }
-    #
-    #     # Added in Cloudera Manager 5.2+
-    #     NAVIGATORMETASERVER { ... }
-    # }
+
+    databaseTemplates {
+        CLOUDERA_MANAGER {
+            name: scmt
+            databaseServerName: rds-mysql-prod1
+            databaseNamePrefix: scm
+            usernamePrefix: scmu
+        }
+
+        ACTIVITYMONITOR {
+            name: amont
+            databaseServerName: rds-mysql-prod1
+            databaseNamePrefix: amon
+            usernamePrefix: amonu
+        }
+
+        REPORTSMANAGER {
+            name: rmant
+            databaseServerName: rds-mysql-prod1
+            databaseNamePrefix: rman
+            usernamePrefix: rmanu
+        }
+
+        NAVIGATOR {
+            name: navt
+            databaseServerName: rds-mysql-prod1
+            databaseNamePrefix: nav
+            usernamePrefix: navu
+        }
+
+      # Available in Cloudera Manager 5.2+
+      NAVIGATORMETASERVER {
+            name: navmst
+            databaseServerName: rds-mysql-prod1
+            databaseNamePrefix: navms
+            usernamePrefix: navmsu
+        }
+    }
 
     #
     # Configuration to override Cloudera Manager package repositories
     #
 
-    # repository: "http://archive.cloudera.com/cm5/redhat/6/x86_64/cm/5.7/"
-    # repositoryKeyUrl: "http://archive.cloudera.com/cm5/redhat/6/x86_64/cm/RPM-GPG-KEY-cloudera"
+    # RHEL 6 repositories are being used here since they are needed for ACCUMULO16.
+    # RHEL 7 will also work if ACCUMULO16 service is not being used.
+    repository: "http://archive.cloudera.com/cm5/redhat/6/x86_64/cm/5.7/"
```

Comment:
I think for consistency with the current release of Director that leaving this alone is best to avoid confusion. We're already talking about updating all of the repo URLs etc. anyway.